app/resources/parashot.json

```diff
@@ -0,0 +1,252 @@
+[
```

Comment:
I was wondering, since you are already using pyluach, which has ```parshios.getparsha(date)```, why do we need a JSON file?

Reply:
the file inclue beside the parasha name in english(like pylucah)
also the hebrew name and link that explain(to each parasha